_pages/CodeOfConduct.html

```diff
@@ -0,0 +1,120 @@
+
+            <p>We also have a set of<a href="{{ site.baseurl }}/participation-guidelines">
+              Participation guidelines</a> for the events events.</p>
+
+
+            <p>Harassment includes, but is not limited to:</p>
+            <ul>
+              <li>Verbal comments that reinforce social structures of domination related to
+                gender, gender identity and expression, sexual orientation, disability,
+                physical appearance, body size, race, age, religion or work experience</li>
+                <li>Sexual images in public spaces</li>
+                <li>Deliberate intimidation, stalking or following</li>
+                <li>Harassing photography or recording</li>
+                <li>Sustained disruption of the Event</li>
+                <li>Uninvited physical contact</li>
+                <li>Unwelcome sexual attention</li>
+                <li>Advocating for, or encouraging, any of the above behaviour</li>
+            </ul>
+
+            <p>Anything that makes someone feel uncomfortable could be deemed
+              harassment.
+            For more information about what constitutes harassment and examples, please
+            refer to <a href="http://www.opencon2017.org/code_of_conduct">
+            OpenCon’s Code of Conduct in Brief</a>
+             and the <a href="http://openhardware.science/gosh-2017/gosh-code-of-conduct/">
+             Gathering for Open Source Hardware’s examples of behaviour</a>
+           </p>
+
+           <h2>Enforcement</h2>
+           <p>Participants (including organisers) asked to stop any harassing
+              behaviour are expected to comply immediately. If a participant
+              engages in harassing behaviour, event organisers retain the right
+              to take any actions to keep the event a welcoming environment for
+               all participants. This includes warning the offender or expulsion
+               from the event.</p>
+
+            <p>Event organisers may take action to redress anything designed to
+              disrupt, or with the clear impact of disrupting, the event or
+              making the environment hostile for any participants.</p>
+
+            <p>We expect participants to follow these rules at all event venues
+              (including online) and event-related social activities.</p>
+
+            <h2>Reporting</h2>
+
+            <p>If someone makes you or anyone else feel unsafe or unwelcome,
+              please report it as soon as possible. Harassment and other Code
+              of Conduct violations reduce the value of our event for everyone.
+               We want you to be happy at our event. People like you make our
+               event a better place.</o>
+
+            <p>You can make a report either personally or anonymously.
+              All reports will be handled with discretion.</p>
+
+            <p>You can make a personal report directly to *****.</p>
```

Comment:
Who do we put here?

Reply:
I would either suggest yourself or a local organiser for each of the workshops